invenio_rdm_records/collections/service.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+"""Collections service."""
+
+from invenio_communities.proxies import current_communities
+
+from .api import Collection, CollectionTree
+
+
+class CollectionItem:
+    """Collection item."""
+
+    def __init__(self, collection: Collection):
+        """Instantiate a Collection object.
+
+        Optionally pass a community to cache its information in the collection's instance.
```

Comment:
This looks copy-pasted from somewhere

Reply:
This code was from my prototype, and then I changed the signature and forgot about the docs.  

Thanks for spotting it!